tuts/start-a-private-network/v2.0.0-alpha.7/keygen.md

```diff
@@ -104,16 +104,3 @@ learning purposes.
 | **ED25519**   |                                                                              |
 | Public key    | `0x0fe9065f6450c5501df3efa6b13958949cb4b81a2147d68c14ad25366be1ccb4`         |
 | SS58 Address  | `5CRZoFgJs4zLzCCAGoCUUs2MRmuD5BKAh17pWtb62LMoCi9h`                           |
-
-### Pair 3
-
```

Comment:
Any particular reason to provide fewer keys? I'm not necessarily opposed to the change, just curious.

Reply:
Because we only use two pairs of keys in this tutorial I think it's a bit confusing to provide three pairs of keys as examples. There's already a lot of different types of keys flying around and I fee like this may make it _less_ clear to the user what the right number of the right types of keys are.